pages/docs/guides/message-validation.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Validate Messages/Events
+description: This guide explains two ways to validate messages/events in AsyncAPI documents.
```

Comment:
```suggestion
title: Validate Messages/Events in runtime
description: This guide explains different use cases for messages/events validation using AsyncAPI.
```

This should be a guide where we just explain different ways of doing things. Now it sounds like the only tools out there are `asyncapi-validator` and event gateway. The goal should not be to promote them and explain how to use them (that for example Nodejs is needed, or that `npm i` must be run. The goal is to highlight that there are many different ways people approach message validation.

you can start with a diagram showing producer-broker-producer and say "blabla validation can occur on any stage of the flow" and then:
- section: Validation inside producer and consumer -> It could start with the same diagram, but somehow marked what components are critical for that section. And you explain that validation of incoming and outgoing messages can be an integral part of the application. Your AsyncAPI file should contain descriptions of payload schemas, so you can read these in your application, and validate messages that are consumed by the application and also produced. -> and then `asyncapi-validator` comes into play, as just one example of how this could be done. That there is Nodejs library that allows you to pass AsyncAPI file to it, and get validation methods. But no need to install nodejs or the package itself, it is a guide that should just highlight one possible reference implementation. 
- section: Validation in a gateway -> It is not so important to show example AsyncAPI file. More important is to show diagram, the same that was in intro, but with additional component - event gateway (this is something you need to consult with @smoya as I do not remember if this gateway is standalone next to broker or rather a side care to the producer/consumer). And you just explain that the advantage is that validation is delegated to central component. And brought explain on our event gateway example how this could work.
- section: Native broker validation -> here another diagram, showing that validation requests go out from broker to `schema registry`. You write that this is common for Kafka broker. In such cases AsyncAPI is not part of the flow, but still plays an important role, because in AsyncAPI document you can still reference the payload schemas stored in the schema registry instead of duplicating them in AsyncAPI document -> example https://github.com/asyncapi/avro-schema-parser#use-them-on-your-asyncapi-document and you can explicitly document where is the schema registry located https://github.com/asyncapi/bindings/tree/master/kafka#server-binding-object

so a bit of refactor is needed

I'm happy to hop on a call if you need me

Reply:
hey, yes so is it possible to get on group call with you and @smoya ? I have questions related to studio and event gateway too plus I need some help with code snippets here.

Reply:
> hey, yes so is it possible to get on group call with you and @smoya ? I have questions related to studio and event gateway too plus I need some help with code snippets here.

Just for the record, this happened today.

Reply:
![image](https://user-images.githubusercontent.com/74637789/198022757-6cc94ee9-6819-462a-9311-6e8fd981c354.png)

Reply:
![image](https://user-images.githubusercontent.com/74637789/198028119-088e2e74-6c17-4838-9519-1d5fde882802.png)

Reply:
![image](https://user-images.githubusercontent.com/74637789/198028990-27a425d6-f1c4-4fd1-862f-55f921edf789.png)

Reply:
![image](https://user-images.githubusercontent.com/74637789/198036894-48a7fbf7-d8a3-4fe6-ac8d-ecc990b5cba1.png)